interface/urls.py

```diff
@@ -10,6 +10,7 @@
     url(r'^repo/(?P<full_name>[\w/.-]+)/delete$', views.RepoDeleteView.as_view(), name='repo_delete'),
     url(r'^repo/(?P<full_name>[\w\-_]+/[\w\-_]+)/(?P<path>[\w/\-._]+)?$',
         views.RepoDetailView.as_view(), name='repo_detail'),
+    url(r'^search/(?P<full_name>[\w\-_]+/[\w\-_]+)', views.search_view, name='search'),
```

Comment:
Oh, I accidentally left out the repo filter. This search is only within a single repo. Cross-repo search would be significantly more complicated, due to permissions being an API interaction. Rob is aware of this restriction, and okay with it.